compiler/rustc_parse/src/lib.rs

```diff
@@ -135,9 +132,8 @@ fn maybe_source_file_to_parser(
     source_file: Lrc<SourceFile>,
 ) -> Result<Parser<'_>, Vec<Diagnostic>> {
     let end_pos = source_file.end_pos;
-    let (stream, unclosed_delims) = maybe_file_to_stream(sess, source_file, None)?;
+    let stream = maybe_file_to_stream(sess, source_file, None)?;
     let mut parser = stream_to_parser(sess, stream, None);
-    parser.unclosed_delims = unclosed_delims;
```

Comment:
Is `Parser::unclosed_delims` field ever populated after this change?

Reply:
Yes, but this field is used in parser itself, I found that we could have more cleanup on `Parser::unclosed_delims`, maybe it's better to do it in another Pr.